modules/UI/toolbars/ToolbarToggler.js

```diff
@@ -34,9 +34,9 @@ function hideToolbar(force) { // eslint-disable-line no-unused-vars
     clearTimeout(toolbarTimeoutObject);
     toolbarTimeoutObject = null;
 
-    if (Toolbar.isHovered()
-            || APP.UI.isRingOverlayVisible()
-            || SideContainerToggler.isVisible()) {
+    if (true !== force &&
```

Comment:
It looks like it was replaced with dockToolbar at some point. Will remove the "force" arg then.